### ISSUE TITLE
NVDA only on latest chromium-based browsers: After a dynamic update, NVDA doesn't read the new content

### ISSUE THREAD

**Issue by gabinortega** [nvaccess/nvda#11980]
After part of the page is updated dynamically, NVDA doesn't read the new content.

This issue is happening always after a particular change on the page.
On other dynamic changes, NVDA behaves ok.
This issue is only happening on the latest versions of chromium-based browsers: Microsoft Edge, Opera, Google Chrome
On Firefox, IE, Legacy Edge, and Chrome version 80.0.3987.149, NVDA behaves ok.

### Steps to reproduce:
1. A portion of the html code is updated dynamically.
2. Navigate to that new content.

### Actual behavior:
  NVDA is silent when the focus is on the new HTML content, just announcing the Page title.
  if the user goes back to the part of the document that was not updated, NVDA reads ok.

### Expected behavior:
  NVDA Should read new dynamic content.

#### NVDA installed/portable/running from source:
  Installed

#### NVDA version:
  2020.1

#### Windows version:
  Microsoft Windows 10 Enterprise
  Version	10.0.18363 Build 18363

#### Name and version of other software in use when reproducing the issue:
  Only latest versions of chromium-based browsers.
  Microsoft Edge, Opera, Google Chrome

#### Does the issue still occur after restarting your computer?
  Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
  No

#### If addons are disabled, is your problem still occurring?
  Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
  Yes

**Comment by feerrenrut** [comment]
Please supply a minimal (html / plain js) test case to demonstrate the issue.

**Comment by bramd** [comment]
I experience similar problems here. I wasn't able to determine the exact condition that triggers this bug, so I can't supply a minimal test case. However, I often encounter it when switching between workspaces in Slack using ctrl+number keys. I'm running the Slack app, which is based on Electron and uses Chrome/Chromium under the hood.

**Comment by cwalshatepic** [comment]
I am also seeing this issue. Specifically, using safeAppend($afe.renderTemplate(template)) in a JavaScript controller to re-render a template on a page based on a user action can include an icon with an alt attribute as specified by WCAG criteria. When a page is re-rendered in this way, navigating to the dynamically-added content with NVDA prevents these icons from being read by NVDA in a Chrome browser (these icons are, however, read out with NVDA in Firefox or with other screen readers in Chrome).

The details in the original bug report apply, except that we are seeing this issue in both Jaws 2020 and Jaws 2021.

**Comment by feerrenrut** [comment]
Without a minimal test sample it is unlikely for this issue to get attention. I'm curious if this can be reproduced in Firefox, if not, it may well be a Chromium bug. In order to get Chromium developers to investigate an fix the issue a minimal test case will be required.

**Comment by Adriani90** [comment]
@gabinortega, @bramd,   @cwalshatepic are you still having this issue in Chrome or Edge 112?

Please test with NvDA last alpha since there was a PR handling some parts of this issue.

You can download the last alpha at this link:
https://www.nvaccess.org/files/nvda/snapshots/

**Comment by Adriani90** [comment]
We didn't get any updates from people who reproduce this since more than two years, I am closing as abandoned. If you are still having this issue, please comment on this and we can reopen.
